department-of-veterans-affairs/va.gov-cms
# VACMS-3757_3758: Change eb perms and cta cardinality.

## Description
See #3757 #3758 

## Testing done
Visual / Behat

## QA steps
- As a content admin (eg randi.hecht@va.gov) go to /node/add/campaign_landing_page
- Try opening all of the new entity browsers (in downloadable resources, events, what you can do)
- Visually verify you do not get access denied message
- In the video panel, add a CTA
- Visually verify you can only add one

### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
